src/components/VButton.vue

```diff
@@ -270,4 +271,8 @@ a.button {
 .grouped-pressed {
   @apply bg-dark-charcoal-10 ring-offset-dark-charcoal-10;
 }
+
+.floating {
+  @apply bg-tx focus-visible:ring-pink;
+}
```

Comment:
We could also name this `plain` instead of `floating`? That might be more generally applicable 🤷

Reply:
Plain might be better. We also called this type of button `.text` at Creative Commons but I think that's a bit confusing.